### PR TITLE
Remove active blank section when adding card

### PR DESCRIPTION
**Pull request by mixonic**
/cc @bantic

### CODE REVIEW

File: tests/integration/components/content-kit-editor/component-test.js [hunk 1]
```diff
@@ -264,6 +264,27 @@ test('it adds a component in display mode to the content-kit editor', function(a
...
+  this.render(hbs`
+    {{#content-kit-editor mobiledoc=mobiledoc cards=cards as |contentKit|}}
+      <button id='add-card' {{action contentKit.addCard 'demo-card'}}></button>
+    {{/content-kit-editor}}
+  `);
+
+  assert.equal(this.$('.content-kit-editor p').length, 1, 'blank section exists');
+  moveCursorTo(this, '.content-kit-editor p');
+  this.$('button#add-card').click();
+  assert.equal(this.$('.content-kit-editor p').length, 0, 'no blank section');
```
Comment (bantic) [inline tests/integration/components/content-kit-editor/component-test.js]: perhaps assert that the card is added as the firstChild of `.content-kit-editor`?

File: tests/integration/components/content-kit-editor/component-test.js [hunk 2]
```diff
@@ -264,6 +264,27 @@ test('it adds a component in display mode to the content-kit editor', function(a
   assert.ok(this.$(`#demo-card-editor`).length, 'Card changed to edit mode');
 });
 
+test('it adds a card and removed an active blank section', function(assert) {
```
Comment (bantic) [inline tests/integration/components/content-kit-editor/component-test.js]: "removes"

**Comment by bantic** [comment]
lgtm

**Comment by mixonic** [comment]
Thanks for the :eyes:

**Pull request merged**